registry/src/storage/tests.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 Helsing GmbH
```

Comment:
Can this file live in "registry/tests/storage.rs"?

Reply:
(Following the rule of IO related tests being declared outside of the source code)

Reply:
For sure, this file is actually unused (I just forgot to delete it).